core/templates/dev/head/pages/state_editor/state_content_editor_directive.html

```diff
@@ -37,4 +47,42 @@
     <button type="button" class="btn btn-default pull-right" ng-click="cancelEdit()">Cancel</button>
     <div style="clear: both;"></div>
   </div>
+  <div ng-if="isCardHeightLimitReached && showCardHeightLimitWarning">
+    <span class="card-height-limit-warning">
```

Comment:
Make it `oppia-card-height-limit-warning`.

Reply:
Done!